examples/http_server/module/src/lib.rs

```diff
@@ -37,15 +37,37 @@ impl Node<Invocation> for StaticHttpServer {
     fn handle_command(&mut self, invocation: Invocation) -> Result<(), OakError> {
         let request = invocation.receive()?;
 
-        info!("Handling a request to {}.", request.path);
+        info!("Handling a request to {}.", request.uri);
 
-        let http_response = HttpResponse {
-            body: "Hello from HTTP server!\n".to_string().into_bytes(),
-            status: 200,
-            headers: hashmap! {},
+        // Workaround for https://rust-lang.github.io/rust-clippy/master/index.html#borrow_interior_mutable_const.
+        static CONTENT_TYPE: http::header::HeaderName = http::header::CONTENT_TYPE;
```

Comment:
Can this be moved outside the function?

Reply:
Possibly yes, but since it's used only here, I think this is fine? Also TBH I don't really understand why clippy complains about it, the type seems pretty simple to me, so I'm not sure why it considers it internally mutable, maybe the macros used to generate the various header names are doing something dodgy internally.

Reply:
I also cannot see where the internal mutability comes from by having a quick look at the code for `HeaderName`.